app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,90 @@
+class GroupMembersController < ApplicationController
+  include CollaboratorProcessing
+
+  before_action :find_group_member, only: [:destroy, :make_admin]
+  before_action :check_admin_member_present, only: :destroy
+
+  def create
+    if group_member_params[:user_ids].present?
+      user_ids = group_member_params[:user_ids].split(',')
+    else
+      flash[:warning] = 'At least one user must be added!'
+      redirect_to group_path(group_member_params[:group_id])
+    end
+
+    if user_ids.present?
+      user_ids.each do |user_id|
+        group_member = GroupMember.new(
+          user_id: user_id,
+          group_id: group_member_params[:group_id]
+        )
+        if group_member.save
+          group_resources(group_member).each do |resource|
+            add_users_as_collaborators(resource, group_member.user.id.to_s, group_member.group.id)
+          end
+        else
+          flash[:warning] = group_member.errors.full_messages.join(', ')
+          redirect_to group_path(group_member_params[:group_id])
+          return
+        end
+      end
+      flash[:notice] = 'Members successfully added!'
+      redirect_to group_path(group_member_params[:group_id])
+    end
+  end
+
+  def destroy
+    if @group_member.destroy
+
+      group_resources(@group_member).each do |resource|
+        collaborator = resource.collaborators.where(user_id: @group_member.user_id).first
+        remove_collaborator(collaborator) if collaborator.present?
+      end
```

Comment:
Good thinking...though I think we're ok because we are removing only one group member at a time, rather than all the group members within a group.  Additionally, we do verify that a collaborator is present before attempting to remove the collaborator from the resource.  I can see both sides...but I'd like to leave it as it is for now.